index.php

```diff
@@ -0,0 +1,7 @@
+<?php
+require 'vendor/autoload.php';
+use TheRealMVP\TeamHydrator;
+
+$data = TeamHydrator::getData();
```

Comment:
FYI don't need to use 'use' if only referencing the class once - can just write inline

Fine though either way